### ISSUE TITLE
Separate tests in multiple files

### ISSUE THREAD

**Issue by artificiala** [PyThaiNLP/pythainlp#224]
Currently, all tests are in the same file ```tests/__init__.py```. Separating into different files would make it easier for contributors to write test cases specifically for a single module/package. The test files could be executed by either ```unittest``` module with test discovery feature i.e. (```python -m unittest discover -s tests```) or other test runners i.e. ```pytest```

For example

### Option 1: by module
```
tests/
    __init__.py
    tag/
        - test_named_entity.py
        - test_locations.py
        - test_pos.py
    soundex/
        - test_lk82.py
        - test_metasound.py
        - test_udom83.py
    tokenize/
        - test_newmm.py
        - test_longest.py
   ...
```

### Option 2: by package

```
tests/
    __init__.py    
    test_tag.py 
    test_soundex.py
    test_tokenize.py
    ...
```

**Comment by bact** [comment]
This is great. I think we can start with the Option 2, and if some packages are really big, we can break them by modules.

**Comment by artificiala** [comment]
Close this issue as it was implemented (see #234)
